scripts/bundles/internal-platform-hydrate.ts

```diff
@@ -51,12 +51,12 @@ export async function internalHydrate(opts: BuildOptions) {
      * import "path";
      * ```
      *
-     * If you say `moduleSideEffects: false` you're basically saying 'dont assume
-     * that imported a module has a desired side effect' which allows an import that
+     * If you say `moduleSideEffects: false` you're basically saying "don't assume
+     * that imported a module has a desired side effect" which allows an import that
      * doesn't result in any symbols being pulled into the importing module to be
      * eliminated.
      *
-     * Another way to say this is that without this change the treeshaking
+     * Another way to say this is that without this change the tree-shaking
```

Comment:
some of these things are sort of "style" questions - most usage around the web seems to be "tree shaking" or "tree-shaking" (eg <https://developer.mozilla.org/en-US/docs/Glossary/Tree_shaking>, <https://rollupjs.org/faqs/#what-is-tree-shaking>) but comparatively little usage of "treeshaking". I went with "tree-shaking" throughout, but with things like this I think generally it should be just like formatting with prettier, i.e. if we decide we want on variant to the exclusion of the others we can use `!tree-shaking` type rules in CSpell to forbid certain variants.